src/client/ui.ts

```diff
@@ -204,12 +204,14 @@ export class Ui implements UiInterface {
 		if (this.options.uiType === 'inline') return
 		this.popupContainer.classList.add('close')
 		this.popupContainer.classList.remove('open')
+		this.client.eventSender('closed-overlay', null)
 	}
 
 	openOverlay() {
 		if (this.options.uiType === 'inline') return
 		// Prevent out-of-popup click from closing the popup straight away
 		setTimeout(() => {
+			this.client.eventSender('opened-overlay', null)
```

Comment:
that should do the work, but just consider analytics lib is only depending on the event as of now, I reckon it can avoid analytics lib coupling with UI/dom if that can be passed as the data of the `opened-overlay` event. 

But if that's too much effort for TN, I'm happy to query that from the dom, probably follow-up question will be, do I query by class `overlay-tn`, is there only one match? cheers